lib/matplotlib/backends/backend_ps.py

```diff
@@ -796,11 +796,12 @@ def _print_ps(
 
         dsc_comments = {}
         if isinstance(outfile, (str, os.PathLike)):
+            filename = os.fspath(pathlib.Path(outfile).name)
```

Comment:
Name is a string, so fspath does nothing and can be removed.

Reply:
Yes, fixed.